util/globals.js

```diff
@@ -14,10 +14,10 @@ function URLParse (url) {
       //path
       //video code
       //query string
-  const pattern = /^((?:https?:)?\/\/)?((?:www|m)\.)?((?:youtube\.com|youtu.be))(\/(?:[\w\-]+\?v=|embed\/|v\/)?)([\w\-]+)(\S+)?$/
+  const pattern = ^(http(s)?:\/\/)?((w){3}.)?youtu(be|.be)?(\.com)?\/.+
```

Comment:
Does this support the capture groups mentioned in the function? We still need to be able to parse the URL into its components. :)